static/etc/udev/rules.d/90-fixrtc.rule

```diff
@@ -0,0 +1,2 @@
+# when rtc* is added run fixrtc
+SUBSYSTEM=="rtc", KERNEL=="rtc*", RUN+= "/usr/bin/fixrtc"
```

Comment:
Fwiw, this should probably *not* live in /usr/bin but something like /usr/lib/core-fixrtc or something less user visible

Reply:
This is moved to /usr/lib/core/fixrtc now.